lib/ziti-tunnel-cbs/ziti_hosting.c

```diff
@@ -34,6 +34,8 @@ limitations under the License.
 #endif
 #endif
 
+#define ZITI_MTU (15 * 1024)
```

Comment:
15k? Does this relate to a ziti or uv parameter?

Reply:
that was Ziti limit

Reply:
i was going to ask if this was the source of the issue olaf reported in discourse. it was just moved from down below though